pkg/virtctl/guestfs/guestfs.go

```diff
@@ -38,9 +38,8 @@ const (
 	diskDir           = "/disk"
 	diskPath          = "/dev/vda"
 	podNamePrefix     = "libguestfs-tools"
-	applianceDir      = "/usr/local/lib/guestfs"
 	guestfsVolume     = "guestfs"
-	appliancePath     = applianceDir + "/appliance"
+	guestfsHome       = "/home/guestfs"
```

Comment:
I update and put the reason in the commit. Basically, I was previously using the same path for the appliance and `HOME`. This was backed by an emptydir. Emptydirs have the nice properties that the permissions are set by k8s based on uid of the pod and are set it at runtime.
However, now I cannot use the same path anymore because the emptydir overwrite the appliance with an empty directory. That's why we cannot use emptydir for the appliance path anymore, but it is still handy for `HOME`

Reply:
Makes sense, thanks for explaining - both here and in the commit message.